web/core/src/transformers/getPokerSteps.js

```diff
@@ -0,0 +1,411 @@
+const PLACEHOLDER_CARD = '2c';
+
+function _getActionStringsFromACPC(bettingString, nextPlayerIndex, numPlayers = 2) {
```

Comment:
Can you remove these from getRepeatedPokerStateForStep just so we don't have duplciated code?